### PR TITLE
Denis/intro

### PR DESCRIPTION
**Pull request by ignaden**
changes to the intro page

### CODE REVIEW

File: notebooks-src/Introduction.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ slug: introduction
...
-- It contains myriad of techniques that are required for "real world" industrial applications of automated reasoning that would typically be implemented in disparate tools.
-- It's cloud-native - to scale to "real-world" applications, we've had to do a lot of engineering which you can now access via APIs and intergrate into your products.
+- It's built on many breakthroughs that eliminate manual work that was historically required (e.g. using academic tools). Traditionally, logical analysis of code for industrial applications required a PhD - with Imandra, you just write code and properties you wish to analyze, and Imandra does the work.
+- It contains a myriad of techniques that are required for "real world" industrial applications of automated reasoning that would typically be implemented in disparate tools.
+- It's cloud-native - to scale to "real-world" applications, we've had to do a lot of engineering which you can now access via APIs and integrate into your products.
 
-![Imandra Core Overview](../assets/../notebook-assets/core-process.png)
+![Imandra Core Overview](../assets/../notebook-assets/imandra-core-process.svg)
```
Comment (mattjbray) [inline notebooks-src/Introduction.md:20]: These relative URLs won't work on the docs site so we need to upload the image to a google cloud bucket and link to that - see https://github.com/imandra-ai/imandra-docs/commit/701b20124bc9ae7a025b37ea7a67cbc51f76b2f5#r122175169

**Review by mattjbray — COMMENTED** [review]
Looks good - just need to upload the SVGs to a gcloud bucket.

File: notebooks-src/Introduction.md [hunk 2]
```diff
@@ -13,11 +13,11 @@ slug: introduction
 
 Imandra is a programming language and an automated reasoning engine which you can use to create models/programs and reason about them. Imandra uses OCaml as the input language - we've given its "pure" subset a mechanized formal semantics allowing Imandra to automatically translate the code into mathematical logic and analyze it using a variety of cutting-edge techniques. Imandra is very special in several important ways, here are just a few:
```
Comment (actionshrimp) [inline notebooks-src/Introduction.md:14]: "Imandra is very special" sounds a bit strange - maybe just remove the last sentence, and change to "analyze it using a variety of cutting-edge techniques:"

File: notebooks-src/Introduction.md [hunk 3]
```diff
@@ -26,15 +26,15 @@ The following pages contain further information on the technical details behind
 
 ## What can you do with Imandra?
 
-The short answer is "Quite a lot!" Our customers have used Imandra to verify complex algorithms, rigorously test and audit their complex software systems, develop new AI product offerings and much more. Imandra brings the most value where/when you rely on algorithms, which is pretty much almost everything these days.
+The short answer is "Quite a lot!"  As discussed above - at its core, Imandra is a unified collection of tools for reasoning about code. Our customers have used Imandra to verify complex algorithms, rigorously test and audit their complex software systems, develop new AI product offerings and much more. Imandra brings the most value where/when you rely on algorithms, which is pretty much almost everything these days.
```
[the remark this answers is not in the document]
Comment (actionshrimp) [reply notebooks-src/Introduction.md:29]: last sentence sounds a bit strange. suggestion: "Imandra adds value and enhances understanding whenever algorithms are relied on as parts of the complex systems that permeate our daily lives"  (?)

File: notebooks-src/Introduction.md [hunk 4]
```diff
@@ -127,11 +127,11 @@ The concept of "digital twins" has become envogue right now, but we've applied t
...
 
-![Model Based Software Development](../assets/../notebook-assets/imandra-mbsd-light.png)
+![Model Based Software Development](../assets/../notebook-assets/imandra-mbsd-dark-extended.svg)
 
 ### Apply formal methods to answer questions about programs (e.g. Neural Networks)
 
-Formal verification seeks to apply local reasoning to answer deep questions about behavior of programs. In our case, programs written in IML. Use of formal methods (including verification) is present in most applications of Imandra, so quite a few of the demos in our gallery will have relevant information and tutorial.
+Formal verification seeks to apply logical reasoning to answer deep questions about behavior of programs. In our case, programs written in IML. Use of formal methods (including verification) is present in most applications of Imandra, so quite a few of the demos in our gallery will have relevant information and tutorials.
 
 [In this example](https://docs.imandra.ai/imandra-docs/notebooks/supervised-learning/ "Analyzing Machine Learnt models with Imandra"), we illustrate how to encode a model of a neural network:
```
Comment (actionshrimp) [inline notebooks-src/Introduction.md:137]: Again, maybe a higher level description of the example, with a link to the notebook containing more detail + code would be enough here, vs a big load of code that might scare off someone who is unfamiliar with OCaml (we talk about learning OCaml at the bottom of the page)?

It is nice to see the example instance output to prove something real is going on though... tricky one.

**Review by actionshrimp — COMMENTED** [review]
A few additional points which Github wouldn't let me inline (I think these parts are already on master?)

- https://github.com/imandra-ai/imandra-docs/blob/b8c96d7135b0de635a3c4b2c1855e9286410e020/notebooks-src/Introduction.md?plain=1#L12
  - style: no quotes needed around "from the beginning"

- https://github.com/imandra-ai/imandra-docs/blob/b8c96d7135b0de635a3c4b2c1855e9286410e020/notebooks-src/Introduction.md?plain=1#L45
  - part in brackets seems out of place
  - IML is introduced here for the first time. Could do with a bit of prior explanation, or at least acronym definition
  - suggestion: '"Raw" use of Imandra Core, where the LLM is trained to trained to translate problems/questions into Imandra Modeling Language (IML) and query Imandra for answers. This can be done via Imandra's HTTP, Python or OCaml APIs.'

- https://github.com/imandra-ai/imandra-docs/blob/b8c96d7135b0de635a3c4b2c1855e9286410e020/notebooks-src/Introduction.md?plain=1#L124
  - 'en vogue'

**Review by actionshrimp — APPROVED** [review]

**Pull request merged**